src/components/data-management/SamplesTable.jsx

```diff
@@ -42,6 +42,10 @@ import 'utils/css/data-management.css';
 const { Paragraph, Text } = Typography;
 
 const exampleDatasets = [
+  {
```

Comment:
instead of having the dataset filename and description hardcoded in the UI, can you:
- modify the UI to simply download and display all datasets in the example datasets s3 bucket
- modify that example datasets s3 bucket to get the description from there.
In that way, we can add and remove new datasets without having to modify the ui, also it will work for multi-account deployments with different datasets in different accounts